gmm_client/src/http_utils.erl

```diff
@@ -33,13 +33,14 @@ get_address(_) ->
 
 -spec build_url(Address :: binary(), Path :: binary()) -> binary().
 build_url(Address, Path) ->
-    << Address/binary, "/", Path/binary >>.
+    binary:replace(<< Address/binary, "/", Path/binary >>, <<" ">>, <<"_">>, [global]).
```

Comment:
How does this work and why is it useful?

Reply:
it's simple - it replaces all spaces with underscores - when there were spaces in urls, errors 505 happened

Reply:
fair enough, but were there any such situations in the first place?

Reply:
Yes, Kamil's generator generates names of some vertices with spaces